templates/components/cart/totals.html

```diff
@@ -21,6 +21,18 @@
             </div>
         </li>
     {{/if}}
+    {{#if cart.fees}}
+        {{#each cart.fees}}
```

Comment:
tested locally, seems without if block, if `cart` doesn't has `fee` key, page won't break. However I fee it's safe to add a pre-check, don't know if it's OK.

Reply:
Looks fine to me